src/cli/referenda.rs

```diff
@@ -784,15 +774,13 @@ async fn refund_submission_deposit(
 	// Create refund_submission_deposit call
 	let refund_call = quantus_subxt::api::tx().referenda().refund_submission_deposit(index);
 
-	let tx_hash = submit_transaction(quantus_client, &keypair, refund_call, None).await?;
+	let tx_hash = submit_transaction(quantus_client, &keypair, refund_call, None, false).await?;
```

Comment:
we could have method overload here instead of adding false everywhere
ie have the old submit_transaction without the parameter, call the new submit_transaction with false.